portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,88 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>A Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
+    
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
+      <!--Header  -->
+      <h1 style="margin:0;padding:0; font-family:'Brush Script MT'; font-size:80px">
```

Comment:
What do you think about adding inline styles like this to your stylesheet?

Reply:
Oh! That makes sense. I created a new `<div>` tag as well. Hopefully this makes it more readable.

`#banner h1 {                               `
`  margin:0;                                   `
`  padding:0;                                 `
`  font-family:'Brush Script MT';    `
`  font-size:80px;                           `
`}                                                   `